src/consortiaServices.js

```diff
@@ -15,3 +15,11 @@ export function checkIfUserInMemberTenant(stripes) {
 
   return stripes.okapi.tenant !== stripes.user.user?.consortium?.centralTenantId;
 }
+
+export function checkIfSharedRecord(stripes, source) {
+  return source.includes('CONSORTIUM-') || checkIfUserInCentralTenant(stripes);

```

Comment:
I think `checkIfSharedRecord` and `checkIfLocalRecord` should be in `ui-inventory` because `source` is specific only to Instance records. Please move it to some utility file in that module

Reply:
The `source` also will be applied in `ui-quick-marc` for authority records. It can be found in `authority-storage/authorities/id`.

Reply:
For `ui-quick-marc` please see [my comment](https://github.com/folio-org/ui-quick-marc/pull/570#discussion_r1279061948)
IMO just two modules using one property is not enough to put this code in `stripes-core` and we can get await with passing `shared` as a prop or even duplicating two small functions
Also, having these utils in `stripes-core` implies that they can potentially be used for any type of record, but that's not the case.